ui/src/components/ResultsTable/ResultsTable.js

```diff
@@ -74,6 +75,9 @@ export class ResultsTable extends Component {
       <Header as="h3" content={title} subheader={subtitle} />
     ) : null;
 
+    if (!(title || subtitle || !isEmpty(headerActionComponent))) {
```

Comment:
I find this more readable, but it might be just me
`if (!title || !subtitle || isEmpty(headerActionComponent)) {`